worker/Dockerfile

```diff
@@ -21,7 +21,7 @@ RUN apk update && apk upgrade && apk add --no-cache --virtual builds-deps build-
 
 RUN apk add jq
 
-RUN python3 -m pip install awscli
+RUN apk add --no-cache aws-cli
```

Comment:
add comment on why we added this change. don't need to justify since we will throw away the code soon. just for documentation.